src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocparamorder/InputJavadocParamOrderCorrect.java

```diff
@@ -0,0 +1,106 @@
+/** A block comment in the syntax tree with no parent. */
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocparamorder;
+
+/** Javadoc for import */
+import java.util.*;
+
+/**
+ * Config: default
+ *
+ * @param <A> A type param
+ * @param <B1> Another type param
+ * @param <C456>> Another type param
+ * @author Nobody
+ * @version 1.0
+ */
+class InputJavadocParamOrderCorrect<A,B1,C456 extends Comparable> {
+
+    /**
+     * The client's first name.
+     * @serial
+     */
+    private String sSecondName;
+
+    /**
+     * Description.
+     *
+     * @version Some javadoc.
+     * @param <T>
+     *            Parameter T desc.
+     * @param p1
+     *            Parameter 1 desc.
+     * @param p2
+     *            Parameter 2 desc.
+     * @param p3
+     *            Parameter 3 desc.
+     * @param p4
+     *            Parameter 3 desc. // OK
+     * @return p1 xyz
+     * @see more javadoc. // OK
```

Comment:
`OK` -> `ok`,  for all occurrences.

Reply:
done

Reply:
Not done. There are still 19 occurrences of `OK` in this PR. Please do not resolve items until I comment "resolved".

Reply:
but this review is outdated now, i have changed OK to ok already.

Reply:
https://github.com/checkstyle/checkstyle/blob/19b10dafa5249c87ca1779289ff4d94ba76cd051/src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocparamorder/InputJavadocParamOrderNoJavadoc.java#L12

https://github.com/checkstyle/checkstyle/blob/19b10dafa5249c87ca1779289ff4d94ba76cd051/src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocparamorder/InputJavadocParamOrderNoJavadoc.java#L29

Reply:
check now please

Reply:
Resolved for test inputs, I would also fix the 12 occurrences in documentation to be consistent.

Reply:
done, please check

Reply:
Resolved.